### PR TITLE
Refactor `inspec` and `yaml` files

### PR DESCRIPTION
**Pull request by myii**
* refactor(inspec): use `os:family` simplification for `config_spec`
* refactor(yaml): sort `default` and `config` alphabetically
* refactor(yaml): move shared values to `defaults.yaml`

---

@n-rodriguez Follows on from #4 as discussed in the last comment there.

My main concern is about the alphabetical sort that's been applied.  Perhaps some of the previous ordering needs to be maintained, such as for the paths?  Please let me know if there are any issues.

### CODE REVIEW

File: rkhunter/osfamilymap.yaml [hunk 1]
```diff
@@ -39,143 +31,126 @@ Debian:
...
-      - /var/log/pki/pki-tomcat/kra/system
-    scriptwhitelist:
-      - /usr/bin/whatis
-      - /usr/bin/ldd
-      - /usr/bin/groups
-      - /usr/bin/GET
-      - /sbin/ifup
-      - /sbin/ifdown
+    allowdevfile:
+      - "/dev/md/autorebuild.pid"
```
Comment (myii) [inline rkhunter/osfamilymap.yaml]: @n-rodriguez By the way, I was planning to remove these quotation marks from the paths, to be consistent throughout, but I thought I'd ask you about them first.

Comment (n-rodriguez) [reply rkhunter/osfamilymap.yaml]: I think they can be safely removed

**Review by n-rodriguez — CHANGES_REQUESTED** [review]

File: rkhunter/osfamilymap.yaml (hunk 1 above)
Comment (myii) [reply rkhunter/osfamilymap.yaml]: Done.

**Review by n-rodriguez — APPROVED** [review]

**Comment by myii** [comment]
@n-rodriguez Thanks for the review.  Fixed the quotes, so the only thing left is if there are any issues with the changes to alphabetic ordering.

**Comment by n-rodriguez** [comment]
> if there are any issues with the changes to alphabetic ordering.

I don't know if it's really an issue... rkhunter is basically a shell script that do some checks with the help of Perl. The `/etc/rkhunter.conf` file is loaded at runtime.

**Comment by myii** [comment]
@n-rodriguez Thanks for the approval.  Do you mind merging this?

**Pull request merged**